### PR TITLE
Eplanning Bid Adapter: add support for video

### PR DESCRIPTION
**Pull request by ChrisHuie**
Resubmit of pr #9020

### CODE REVIEW

**Comment by ChrisHuie** [comment]
@fndigrazia looks like there are some unit tests failing if you can check.

**Comment by fndigrazia** [comment]
Hello @ChrisHuie, thanks for creating the pr. The tests failed because you have to delete line 121 "ad: ad.adm,". In the original pr that line was deleted.

**Comment by ChrisHuie** [comment]
> Hello @ChrisHuie, thanks for creating the pr. The tests failed because you have to delete line 121 "ad: ad.adm,". In the original pr that line was deleted.

Fixed. Thanks 🙏

File: modules/eplanningBidAdapter.js [hunk 1]
```diff
@@ -236,17 +252,42 @@ function getSpacesStruct(bids) {
...
+
+  return (Array.isArray(size) && size.length == 2) ? size : undefined;
+}
+
 function cleanName(name) {
   return name.replace(/_|\.|-|\//g, '').replace(/\)\(|\(|\)|:/g, '_').replace(/^_+|_+$/g, '');
 }
 
 function getSpaces(bidRequests, ml) {
+  let impType = bidRequests.reduce((previousBits, bid) => (bid.mediaTypes && bid.mediaTypes[VIDEO]) ? (bid.mediaTypes[VIDEO].context == 'outstream' ? (previousBits | 2) : (previousBits | 1)) : previousBits, 0);
```
Comment (osazos) [inline modules/eplanningBidAdapter.js:270]: Not required, but for better readability, would you want to replace the part
`bid.mediaTypes[VIDEO].context == 'outstream' ? (previousBits | 2) : (previousBits | 1)`  

by 

`bid.mediaTypes[VIDEO].context == 'outstream' ? (previousBits | VAST_OUTSTREAM) : (previousBits | VAST_INSTREAM)`

**Review by osazos — APPROVED** [review]
LGTM

**Pull request merged**